integration-test/src/main/java/com/sequenceiq/it/cloudbreak/context/TestContext.java

```diff
@@ -1133,6 +1139,29 @@ public void handleExceptionsDuringTest(TestErrorLog testErrorLog) {
             String errorMessage = errorLogMessageProvider.getMessage(exceptionMap, clues);
             testErrorLog.report(LOGGER, errorMessage);
 
+            ITestResult testResult = Reporter.getCurrentTestResult();
+            Throwable testFailException = errorLogMessageProvider.getException(exceptionMap);
+
+            if (testFailException != null) {
+                testResult.setThrowable(testFailException);
+                testResult.setTestName(getTestMethodName().get());
+                testResult.setStatus(2);
```

Comment:
What does status 2 mean? Why 2 exactly?

Reply:
This means [ITestResult.FAILURE](https://www.javadoc.io/doc/org.testng/testng/6.8.21/constant-values.html#org.testng.ITestResult.FAILURE)

Here we have to set the correct test results for test, because of the our TestContext is not [ITestContext](ITestContext) indeed. So our project generates all the errors and related messages as Reporter Output.

Reply:
So, can we use the Enum from there, or not a variable that is more descriptive?